meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -368,6 +377,21 @@ python() {
 # Helpers and internal variables.
 ################################################################################
 
+# @return the full path to the uboot environment device (IE /dev/mmcblkXbootY or /dev/mmcblkX)
+def mender_get_uboot_env_mmc_linux_device_path(d):
+    # Convert u-boot partition numbers to linux device references
+    # See [CONFIG_SYS_MMC_ENV_PART](https://github.com/u-boot/u-boot/blob/u-boot-2016.09.y/README#L4390) and
+    # [linux kernel mmc-dev-parts documentation](https://www.kernel.org/doc/Documentation/mmc/mmc-dev-parts.txt)
+    storage_device=d.getVar("MENDER_STORAGE_DEVICE",True)
+    env_part=d.getVar("MENDER_UBOOT_CONFIG_SYS_MMC_ENV_PART",True)
+    env_part_append=""
+    if env_part != "0":
+        env_part_append="boot%d" % (int(env_part)-1)
+    return storage_device + env_part_append
+
+# Does not include the /dev prefix
+MENDER_UBOOT_MMC_ENV_LINUX_DEVICE_PATH ?= "${@mender_get_uboot_env_mmc_linux_device_path(d)}"
```

Comment:
You will need to add this one also to `mender-vars.json`.

Reply:
Done, thanks